privacy.md

```diff
@@ -193,8 +193,8 @@ right to erase your data:
 # Contacting us
 
 If you have any questions about this policy or our use of your data, please
-feel free to [contact us](privacy@community.theforeman.org) or on
-[IRC](/support.html#IRClivechat).
+feel free to [contact us](mailto:privacy@community.theforeman.org) or on
+[IRC](/support.html#liberachat).
```

Comment:
Perhaps instead we should revert the header back to not break links?

Reply:
There's only two more instances of `IRClivechat`:

```
_includes/manuals/2.4/1.2_release_notes.md:62:* Update all freenode references to point to https://theforeman.org/support.html#IRClivechat or libera.chat ([#32809](https://projects.theforeman.org/issues/32809))
_includes/manuals/2.5/1.2_release_notes.md:93:* Update all freenode references to point to https://theforeman.org/support.html#IRClivechat or libera.chat ([#32809](https://projects.theforeman.org/issues/32809))
```
I've changed those as well.

Reply:
What I meant (and still think) we should do is to align on `IRClivechat` and change the header on the support page to match. My reasoning is that we use IRC which happens to be libera.chat now but as we've seen, we can switch.

Reply:
OK, I've reverted using `liberachat' . However, I am unsure if it works using `{#IRClivechat}` as an additional anchor, as it should be based on the level 2 heading in `support.md`.